src/app_service/service/transaction/service.nim

```diff
@@ -209,6 +214,21 @@ QtObject:
     )
     self.threadpool.start(arg)
 
+
+  proc onFetchDecodedTxData*(self: Service, response: string) {.slot.} =
+    let data = parseJson(response)
+    self.events.emit(SIGNAL_TRANSACTION_DECODED, TransactionDecodedArgs(dataDecoded: data["result"].getStr, txHash: data["txHash"].getStr))
```

Comment:
@Cuteivist if you look at `fetchDecodedTxDataTask` you can see there is a case where received `response` here may be an empty string, as well we can even get in some cases an empty string from the status-go side, hence a direct property access like here (`data["result"]`, `data["txHash"]`) will crash the app. Check for an existence first.

Reply:
Good point. Added try catch and additional check inside